src/components/StorePage.vue

```diff
@@ -40,39 +39,9 @@
           <q-tab default slot="title" name="Products" label="Products" class="text-bold text-tertiary"/>
           <q-tab slot="title" name="Information" label="Information" class="text-bold text-tertiary"/>
           <!-- Targets -->
-          <q-tab-pane name="Products">
-            <q-search clearable inverted :debounce="0" placeholder="Search for products in store!" v-model="search"></q-search>
```

Comment:
Should have just commented it out, this same mark up will be used to display T2s